change/@ni-nimble-components-2376f915-51a5-4555-9c1d-f28c1ca507b1.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add Blazor support for the nimble-card component",
```

Since you haven't modified anything other than the component-status story, you can mark this with a change type of `none`.

```suggestion
  "type": "none",
  "comment": "Update the Blazor status of the nimble-card component",
```